.github/workflows/codegen.yml

```diff
@@ -0,0 +1,19 @@
+name: "SDK Code Generation"
+on:
+  schedule:
+    - cron: '20 23 * * *'
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  generate_and_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Generate and PR
+        uses: algorand/generator/.github/actions/sdk-codegen/
```

Comment:
why hide the action's code in a dot-directory?

Reply:
I just wanted to keep all github related things together in one directory. I could split off everything into a base level directory. 

I think it probably also makes sense to add a Readme to the action directory giving more context on the pieces, so there's some documentation around the whole setup missing probably.